CMakeLists.txt

```
@@ -4,9 +4,18 @@ project(Polars C CXX)
 
 set(CMAKE_CXX_STANDARD 14)
 
-# Dependencies
-set(DEPENDENCIES_DIR "dependencies")
+option(WITH_SUBMODULE_DEPENDENCIES "Use submodules for dependencies" ON)
+option(WITH_TESTS "Build polars_cpp_test target" ON)
+
+if(WITH_SUBMODULE_DEPENDENCIES)
+  # Dependencies
+  set(DEPENDENCIES_DIR "dependencies")
+  include(${DEPENDENCIES_DIR}/CMakeLists.txt)
```

Since `DEPENDENCIES_DIR` is only used here, maybe inline it?

```suggestion
  include(dependencies/CMakeLists.txt)
```

---

👍 makes sense. 
`DEPENDENCIES_DIR` is also used in `dependencies/CMakeLists.txt` so I'll do it there as well